content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -240,9 +240,6 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIStorageCapacity` | `true` | GA | 1.24 | - |
 | `CSRDuration` | `true` | Beta | 1.22 | 1.23 |
 | `CSRDuration` | `true` | GA | 1.24 | - |
-| `ControllerManagerLeaderMigration` | `false` | Alpha | 1.21 | 1.21 |
-| `ControllerManagerLeaderMigration` | `true` | Beta | 1.22 | 1.23 |
-| `ControllerManagerLeaderMigration` | `true` | GA | 1.24 | - |
```

Comment:
These lines (along with the description of the gate) should be moved to `content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md` page rather than being removed. There will always be questions about such **removed** gates. We keep them on a separate page now.

Reply:
> These lines (along with the description of the gate) should be moved to `content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md` page rather than being removed. There will always be questions about such **removed** gates. We keep them on a separate page now.

https://github.com/kubernetes/website/blob/dev-1.26/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md is not there.

I added it. Not sure if it is right.